app/policies/admin_policy.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AdminPolicy
```

Comment:
what is the purpose of this class? it's duplicating `ApplicationPolicy`

Reply:
remove this file